spec/resty/resolver_spec.lua

```diff
@@ -179,11 +179,12 @@ nameserver 127.0.0.1
     end)
 
     it('returns nameserver touples', function()
```

Comment:
I think this deserves a new test.

Reply:
I agree. Also, I think that we have similar problems with other tests. For example, this test is checking that commented lines are ignored, but there isn't an `it` clause explaining that or an explicit assertion.

Reply:
done in e2755eb835e200bf682cd2720f74fe609cb4046d